shared-data/contrib/autoajax/autoajax.js

```diff
@@ -133,15 +133,6 @@ prepare_new_content = function(selector) {
             });
         }
     });
-    var $form = $(selector).find('form#form-search');
```

Comment:
Autoajax only interferes on submit. This means that if autoajax is not present (it is an optional thing at the moment), a normal submit happens and everything still works. Treating autoajax as optional forces certain coding practices which increase the odds that things will actually have a chance of working with Javascript completely disabled. So please leave that mechanism intact!